apps/web/components/chat.tsx

```diff
@@ -0,0 +1,83 @@
+'use client'
+
+import React, { useRef, useState } from 'react'
+import { MessageCircle } from 'lucide-react'
+import { Message } from '@/app/room/[slug]/page'
+import { Button } from './ui/button'
+import { Input } from './ui/input'
+import { ScrollArea } from './ui/scroll-area'
+
+interface ChatProps {
+  messages: Message[]
+  onSend: (message: string) => void
+}
```

Comment:
Replace this interface for a type

```suggestion
type ChatProps =  {
  messages: Message[]w
  onSend: (message: string) => void
}
```

Reply:
https://www.totaltypescript.com/type-vs-interface-which-should-you-use